pkg/frontend/transport/handler.go

```diff
@@ -112,12 +113,18 @@ func NewHandler(cfg HandlerConfig, roundTripper http.RoundTripper, log log.Logge
 			Help: "Number of chunks fetched to execute a query.",
 		}, []string{"user"})
 
+		h.queryIndexBytes = promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Name: "cortex_query_fetched_index_bytes_total",
+			Help: "Number of TSDB index bytes fetched to execute a query.",
```

Comment:
[nit] Contrary to chunk bytes, this is just counted in the store-gateway. I would clarify it.

```suggestion
			Help: "Number of TSDB index bytes fetched from store-gateway to execute a query.",
```